packages/node_modules/pouchdb-checkpointer/src/index.js

```diff
@@ -73,12 +73,15 @@ function updateCheckpoint(db, id, checkpoint, session, returnValue) {
 }
 
 class CheckpointerInternal {
-  constructor(src, target, id, returnValue, opts) {
+  constructor(src, target, id, returnValue, opts = {
+    writeSourceCheckpoint: true,
+    writeTargetCheckpoint: true,
+  }) {
```

Comment:
I don't think this defaults if a value is provided for opts, but one or other of these values aren't provided, eg: `CheckpointerInternal(...., {})`

Reply:
Okay, I've added fallback values for individual properties.
Now you can provide partial options, pass an empty object, pass `undefined`, call without opts, or pass any arbitrary object (except null and frozen/read-only objects).